apps/api/src/admin/participant_manager.py

```diff
@@ -47,6 +47,35 @@ async def get_hackers() -> list[Participant]:
     return [Participant(**user, **user["application_data"]) for user in records]
 
 
+async def get_non_hackers() -> list[Participant]:
+    """Fetch all non-hackers participating in the event."""
+    records: list[dict[str, Any]] = await mongodb_handler.retrieve(
+        Collection.USERS,
+        {
+            "$and": [
+                {"role": {"$exists": True}},
+                {
+                    "role": {
+                        "$not": {
+                            "$regex": "applicant|"
+                            + "|".join(
+                                (
+                                    Role.DIRECTOR,
+                                    Role.REVIEWER,
+                                    Role.CHECKIN_LEAD,
+                                    Role.ORGANIZER,
+                                )
+                            )
+                        }
+                    }
+                },
```

Comment:
Was there a particular reason you chose to use a exclusive filter here? Think it would be simpler to just get the exact roles we need.